src/ResourceManager/Profile/Commands.Profile/Context/GetAzureRMContext.cs

```diff
@@ -26,6 +27,26 @@ namespace Microsoft.Azure.Commands.Profile
     [OutputType(typeof(PSAzureContext))]
     public class GetAzureRMContextCommand : AzureRMCmdlet
     {
+        /// <summary>
+        /// Gets the current default context.
+        /// </summary>
+        protected override AzureContext DefaultContext
```

Comment:
@cormacpayne are you planning to add any tests for this?
Can you also point what version of PS you have it on your machine?
The reason I ask is, I was trying to find the enum on my machine that you have used 'AuthenticationError', I could not find that enum.